### DISCUSSION TITLE
Can I use normal introspection with URQL?

### DISCUSSION THREAD

**Discussion by nandorojo** [dotansimha/graphql-code-generator#7325] [Q&A] [answered]
The `urql-introspection` plugin doesn't have a `minify` option like the regular `introspection` plugin has. Will the normal `introspection` plugin work with URQL? What are the differences? Thanks!

**Comment by ardatan** [comment, marked as answer]
urql-introspection plugin doesn't have everything from introspection but some part of it needed by urql. You can see Urql's documentation for more information: https://formidable.com/open-source/urql/docs/graphcache/schema-awareness/
